app/controllers/host_reports_controller.rb

```diff
@@ -29,8 +51,8 @@ def show
             reported_at: @host_report.reported_at,
           },
           permissions: {
-            can_delete: authorized_for(auth_object: @host_report, authorizer: authorizer, permission: "destroy_#{controller_permission}"),
-            can_view: authorized_for(auth_object: @host_report, authorizer: authorizer, permission: "view_#{controller_permission}"),
+            can_delete: can_delete(@host_report),
+            can_view: can_view(@host_report),
```

Comment:
nit: in Ruby if the method returns only `true` or `false` it's a good style to use `?` in the method's name.